src/tools/dart-compile.md

```diff
@@ -74,39 +75,43 @@ The following table shows the subcommands of `dart compile`.
   <tr>
     <td> <code>exe</code> </td>
     <td> <span style="white-space: nowrap">Self-contained</span> executable </td>
-    <td> A standalone, architecture-specific executable file.
-      <a href="#exe">Learn more.</a>
+    <td> A standalone, architecture-specific executable file containing the source code
+      compiled to machine code and a small <a href="/overview#runtime">Dart runtime</a>.
+      <br><i><a href="#exe">Learn more.</a></i>
     </td>
   </tr>
   <tr>
     <td style="white-space: nowrap"> <code>aot-snapshot</code> </td>
-    <td style="white-space: nowrap"> AOT snapshot </td>
-    <td> An architecture-specific file with <b>no Dart runtime</b>.
-      <a href="#aot-snapshot">Learn more.</a>
+    <td style="white-space: nowrap"> AOT module </td>
+    <td> An architecture-specific file, containing the source code
+      compiled to machine code, and <b>no Dart runtime</b>.
+      <br><i><a href="#aot-snapshot">Learn more.</a></i>
     </td>
   </tr>
   <tr>
     <td> <code>jit-snapshot</code> </td>
-    <td> JIT snapshot </td>
+    <td> JIT module </td>
     <td> An architecture-specific file with
-      parsed classes and compiled code that's generated during
-      a training run of the program.
-      <a href="#jit-snapshot">Learn more.</a>
+      an intermediate representation of all source code,
+      and an optimized representation of the source code
+      exercised during a training run of the program.
```

Comment:
I'm not sure the use of "exercised" will be super clear here, could it simply say "**generated** during a training run..."?

```suggestion
      generated during a training run of the program.
```

Reply:
That changes the meaning quite a bit: The important part here is the only the code that is actually exercised/run by the training input is optimized. Got a better way to phrase that?

Reply:
I see what you mean, maybe "reached" or "executed" instead of "exercised"?

Reply:
Yeah, I used "exercised" as it seemed a good pairing with "training input".